pages/search.js

```diff
@@ -56,11 +56,11 @@ export default function Search({
       {state.context.loading ? (
         <CustomCircularProgress />
       ) : (
-        <Paper className={styles.cars}>
+        <Paper data-cy='search-page-cars' className={styles.cars}>
```

Comment:
Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7125704)
```suggestion
        <Paper data-cy="search-page-cars" className={styles.cars}>
```